Applications/Games/Total War Rome II/Steam/script.js

```diff
@@ -0,0 +1,22 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+include(["Engines", "Wine", "Verbs", "d3dx9"]);
+include(["Engines", "Wine", "Verbs", "vcrun2005"]);
+include(["Engines", "Wine", "Verbs", "vcrun2008"]);
+include(["Engines", "Wine", "Verbs", "vcrun2010"]);
+
+new SteamScript()
+    .name("Total War: ROME II")              
+    .editor("Creative Assembly")    
+    .author("ImperatorS79")  
+    .appId(214950)
+    .wineVersion(LATEST_STAGING_VERSION)
+    .wineDistribution("staging")
+    .postInstall(function(wine, wizard) {
+	wine.d3dx9();
+	wine.vcrun2005();

```

Comment:
Does it really need all vcrun versions?

Reply:
Impossible to say from the appdb or the discussion I had with a user of this game.